docs/noise.ipynb

```diff
@@ -62,196 +62,673 @@
-    "`_channel_(self) -> Sequence[np.ndarray]:` method. \n",
-    "\n",
-    "This method should return the sequence of `numpy` matrices corresponding to the Kraus operators. The basis in which this matrix is expressed is always implicit with respect to the object being called. \n",
-    "\n",
-    "For example, in `GateOperations`, these matrices must be ordered with respect to the list of qubits that the channel is applied to. The qubit-to-amplitude order mapping matches the ordering of `numpy.kron(A, B)`, where `A` is a qubit earlier in the list than the qubit `B`.\n",
+    "Cirq defines many commonly used quantum channels in [`ops/common_channels.py`](https://github.com/quantumlib/Cirq/blob/master/cirq/ops/common_channels.py). For example, the single-qubit bit-flip channel\n",
     "\n",
-    "If one has defined `_channel_`, then that `Gate` and any `GateOperation`\n",
-    "that uses that gate can be used as an argument to `cirq.channel` and\n",
-    "`cirq.channel` will return this sequence of matrices.\n",
-    "\n",
-    "Besides objects that support `_channel_`, `cirq.channel` will also fall\n",
-    "back to other objects that can be interpreted as channels. For example, if a channel is a probabilistic mixture of unitary gates (see below), then `cirq.channel` will fall back to seeing if the object supports `_mixture_`. If `_mixture_` is not supported, then `cirq.channel` checks to see if `_unitary_` is supported.\n",
+    "$$\n",
+    "\\rho \\rightarrow (1 - p) \\rho + p X \\rho X\n",
+    "$$\n",
     "\n",
-    "In addition to supporting `_channel_`, objects that are channels should also implement `_has_channel_(self) -> bool` to return `True`. This method is used to determine whether an object has a `_channel_` or not without having to do the potentially expensive creation of the matrices for the channel."
+    "with parameter $p = 0.1$ can be created as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "386a49be9dc7"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"Get a single-qubit bit-flip channel.\"\"\"\n",
+    "bit_flip = cirq.bit_flip(p=0.1)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "01cfc64ceb53"
+    "id": "f1f501177e53"
    },
    "source": [
-    "### `cirq.mixture` and `def _mixture_`\n",
-    "\n",
-    "Some channels can be interpreted as probabilistically selecting between\n",
-    "different unitary evolutions:\n",
-    "\n",
-    "$$\\rho \\rightarrow \\sum_k p_k U_k \\rho U_k^\\dagger {\\rm ~where~} \\sum_k p_k =1 {\\rm ~and~ U_k U_k^\\dagger= I}$$\n",
-    "\n",
-    "In this case, it is possible to perform **Monte Carlo simulations** of these gates using a wave function based simulator (and not a density matrix based simulator). An example of this can be seen below: \n"
+    "To see the Kraus operators of a channel, the `cirq.channel` protocol can be used."
```

Comment:
we could also link to the Protocols page from somewhere - maybe here or in a new sentence for users who don't understand what a protocol is.

Reply:
Sounds good. Is there a standard protocol (ha ha ha) to link to docs from docs? Should it be a devsite path like /cirq/protocols or link to GitHub/elsewhere?

Reply:
If you use the full API symbol in backticks it will autolink to the reference page on the site.

For narrative docs, if in the same project/repo you can relative link to the file (w/ extension), like `[foo guide](./foo_guide.ipynb)`.

But if linking to a doc in another project/repo, need the absolute site path. This keeps the links working in GitHub and Colab.